.all-contributorsrc

```diff
@@ -109,6 +109,17 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "jxd1337",
+      "name": "jxd1337",
+      "avatar_url": "https://avatars.githubusercontent.com/u/138708600?v=4",
+      "profile": "https://github.com/jxd1337",
+      "contributions": [
+        "code",
+        "ideas",
+        "doc"
+      ]
```

Comment:
I think this should be automated - I recall mentioning the bot would automatically create a PR. See #69 

So this change is not really needed :)